### PR TITLE
Added docs for lubuntu 17.10

### PR DESCRIPTION
**Pull request by stealthanthrax**
#6 
I have added docs and release notes for lubuntu 17.10

Please have a look at the format

I have referenced these notes: https://wiki.ubuntu.com/ArtfulAardvark/ReleaseNotes/Lubuntu

### CODE REVIEW

File: releases/lubuntu1710/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<hr>
+
+## Get Lubuntu
+Head over [here](downloads.md) to get it.
+<hr>
+
+## Support
+This release is supported until July of 2018.
+<br>
+[Contact us](http://lubuntu.me/links/).
```
Comment (mariobehling) [inline releases/lubuntu1710/release_notes.md]: Please take out "Contact us"

**Comment by mariobehling** [comment]
I also changed the issue slightly. Please also help with this change:

Use the following structure:
* e.g. ` /releases/lubuntu1710_release_notes.md` and ` /releases/lubuntu1710_downloads.md` and put the md files into the folder following this structure.

**Comment by stealthanthrax** [comment]
done @mariobehling

**Comment by mariobehling** [comment]
Expected is: releases/lubuntu1710_downloads.md

**Comment by stealthanthrax** [comment]
@mariobehling , I can't understand the second query.

**Comment by mariobehling** [comment]
Please check the folder structure. Expected is: `/releases/lubuntu1710_downloads.md`

**Comment by mariobehling** [comment]
ok, I will change it myself.

**Pull request merged**